src/scripts/tray.js

```diff
@@ -87,23 +87,34 @@ function createAppTray () {
     };
 }
 
+function setImage (title) {
+    if (title === '•') {
+        title = "Dot";
+    } else if (!isNaN(parseInt(title)) && title > 9) {
+        title = "9Plus";
+    }
+
+    const _iconPath = path.join(__dirname, 'images', icons[process.platform].dir, `icon-tray${title}.png`);
```

Comment:
Sorry I messed up something with commits. Icons are the same because im not good at graphic someone should change it. They should have diferent sizes.